contents/julia_basics.md

```diff
@@ -2363,3 +2360,260 @@ sco(s; process=catch_show)
 
 > **_NOTE:_**
 > For more complex HTTP interactions such as interacting with web APIs, see the [`HTTP.jl` package](https://github.com/JuliaWeb/HTTP.jl).
+
+### Project Management {#sec:project_management}
+
+One last thing from Julia's standard library for us to cover is the **`Pkg` module**.
+As described in @sec:programmers,
+Julia offers a **native project management solution**,
+with dependencies and version control tightly controlled, manageable, and replicable.
+
+Unlike traditional package managers,
+which install and manage a single global set of packages,
+Julia's package manager is designed around "environments":
```

Comment:
Can we stick to one sentence per line to avoid moving text around when comma's are added or removed?

Reply:
https://sembr.org/

Take a look at 5 there.